### PR TITLE
test: Add target for the test report as html

### PR DESCRIPTION
**Pull request by afbjorklund**
Just something I did during lunch, easier to read than the JSON

`go get -u github.com/vakenbolt/go-test-report`

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -374,6 +374,11 @@ out/test-report.json: $(SOURCE_FILES) $(GOTEST_FILES)
 	-coverprofile=out/coverage.out -json > out/test-report.json
 out/coverage.out: out/test-report.json
 
+# Generate go test report (from gotest) as a a HTML page
+test-report.html: out/test-report.json
```
Comment (medyagh) [inline Makefile:378]: could we call it unit-test-report.html ?

File: .gitignore [hunk 2]
```diff
@@ -22,6 +22,7 @@ _cgo_export.*
 
 _testmain.go
 
+test-report.html
```
Comment (medyagh) [inline .gitignore:25]: how about we generate this in the /out/unit-test-report.html ?

File: Makefile (hunk 1 above)
Comment (medyagh) [reply Makefile:378]: I wanna ensure this wont confuse the users of 
```
$ make help | grep html
html_report                    Generate HTML  report out of the last ran integration test logs.
```

Comment (medyagh) [reply Makefile:378]: or you could make the "html_report" convert all tests to html (including last integraiton and unit test)
that way we dont have to remember which one to call...

jsut ask makefile to give us html for all tests in out/unit-test.html

**Review by medyagh — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (afbjorklund) [reply Makefile]: Renamed and moved to out

**Review by medyagh — APPROVED** [review]
Looks good to me, Thank you for adding more useful unit test report

**Pull request merged**